packages/coordinator/src/services/verification/signature-verification.ts

```diff
@@ -32,6 +32,28 @@ function verifyPrescriptionSignature(parentPrescription: hl7V3.ParentPrescriptio
   return errors
 }
 
+function verifyChain(x509Certificate: crypto.X509Certificate): boolean {
+  const subCACerts = getSubCaCerts()
+  return certIsTrusted(x509Certificate, subCACerts)
+}
+
+const getSubCaCerts = (): Array<string> => {
```

Comment:
could be replaced with a simple line as split() will always return an array.

Reply:
great point. done